### ISSUE TITLE
Easy Multi Column Notes bugs out with inline contexts from Make.md

### ISSUE THREAD

**Issue by g-adel** [zamsyt/obsidian-snippets#9]
I really love this snippet... Unfortunately I can't use it at the moment simply because I really rely on Make.md inline context view (it basically gives a nice interface for the yaml and file name at the top). I was wondering if there was an easy fix from your side about this?

The most obvious bug is that the on the right side you can't really select the text with the mouse near the top of the column. There are other ones that I can list in case you're interested. Thanks :D

**Comment by zamsyt** [comment]
First of all, looks like Make.md needs this to make the header full-width in Live preview:
```css
.mk-header { column-span: all; }
```
Mouse navigation can be buggy anyway, but see if that helps.

**Comment by j-maffe** [comment]
Yup! That totally worked! I'll see if there's any other weird behavior but so far it's all good! Thanks a lot :D

**Comment by zamsyt** [comment]
> I've shared this issue with the Make.md author, hopefully, he adds this change.

Eh, this is a quite niche use case, it's probably not sustainable to ask all weird use cases to accommodate all others. I've been thinking about ways to make customizability and handling of special cases easier/more reliable, but for now I might add this to the snippet.

**Comment by j-maffe** [comment]
Yeah... but this one seemed to be helpful for most interactions with other plugins/snippets. But yeah either way it's definitely harmless to add it to the snippet like I did :D
